lib/fhmongodb.js

```diff
@@ -153,13 +153,19 @@ Database.prototype.tearUp = function (auth) {
           if (interval) {
             clearInterval(interval);
           }
-          self.db = db.db(self.name);
```

Comment:
discussed with @PhilipGough over IRC, this scenario had been covered by tests he ran locally.